fiber/client.py

```diff
@@ -47,6 +47,45 @@ def proto_to_tx(proto: eth_pb2.Transaction):
     tx.s = encode_hex(proto.s)
     return tx
 
+class Block:
+    hash: str
+    number: int
+    parent_hash: str
+    timestamp: int
+    producer: str
+    base_fee_per_gas: int
+    extra_data: str
+    fee_recipient: str
+    gas_limit: int
+    gas_used: int
+    logs_bloom: bytes
+    prev_randao: bytes
+    receipt_root: bytes
+    state_root: bytes
+    transactions: list[Transaction]
```

Comment:
I think for transactions, it gets a bit expensive if we loop and convert every single one of them. I think we should have another method `subscribe_new_headers` that does the same as `subscribe_new_blocks` but without the conversion. Wdyt?

Reply:
I think that makes sense!